packages/terra-modal-manager/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "terra-modal-manager",
+  "main": "lib/index.js",
+  "version": "0.1.2",
+  "description": "A Redux-backed Container component that dynamically presents components in a Terra Modal",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-modal-manager",
+    "ModalManager",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
+    "react-redux": "^5.0.4",
+    "redux": "^3.6.0",
+    "terra-app-delegate": "^0.x",
+    "terra-toolkit": "^0.5.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-redux": "^5.0.4",
+    "redux": "^3.6.0",
+    "terra-base": "^0.x",
+    "terra-app-delegate": "^0.x",
+    "terra-mixins": "^1.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^0.x",
+    "terra-slide-group": "^0.x",
+    "terra-mixins": "^1.0.0",
+    "terra-modal": "^0.3",
```

Comment:
"^0.3" will only resolve patch updates. We should probably change this to "^0.x". This is an issue we will have until we have a 1.0 release out. 

<img width="1236" alt="screen shot 2017-06-27 at 8 55 47 am" src="https://user-images.githubusercontent.com/8146159/27591179-894ded72-5b16-11e7-8f46-4312e0d039d9.png">


https://semver.npmjs.com/